### DISCUSSION TITLE
有序列表都被渲染成无序列表了

### DISCUSSION THREAD

**Discussion by wgx-dh** [YunYouJun/hexo-theme-starry#338] [Q&A 疑难杂症] [answered]
这是我的环境
![image](https://user-images.githubusercontent.com/63238251/185781640-7469084b-9c4c-4d6b-a8d8-6d4260854860.png)
我的博客 markdown 文本是在 linux 环境下编辑的。
问一下大佬，这可能是什么原因？
我印象里 之前在windows 环境下编辑的 markdown 是可以正确渲染的

**Comment by YunYouJun** [comment]
是否有仓库可复现链接？

有序列表和无序列表指的是文章内容？
```md
- a
- b
- c

1. a
2. b
3. c
```

**Reply by wgx-dh** [reply]
仓库地址：https://github.com/WANG-Guangxin/test
这是渲染之后的 html
https://github.com/WANG-Guangxin/test/blob/master/public/posts/20220222/index.html

**Reply by wgx-dh** [reply]
html 标签确实不一样 
但是显示的就是一样的 
![image](https://user-images.githubusercontent.com/63238251/185793481-e0b03d01-2997-453e-a836-79fde01a431e.png)

**Comment by wgx-dh** [comment]
蹲一个回复

**Comment by wgx-dh** [comment]
是我 css 的问题吗

**Comment by wgx-dh** [comment, marked as answer]
是我自己的问题，不知道哪里改错了，换了一个主题版本就好了。

**Reply by YunYouJun** [reply]
版本号的区别是？

**Reply by wgx-dh** [reply]
1.9.1 是有序号的
1.10.6 还是没有序号
我也不知道为什么

**Reply by wgx-dh** [reply]
![image](https://user-images.githubusercontent.com/63238251/188254127-59009bae-8758-4232-9227-cac3292dd44a.png)
你的博客里 也是这样

**Comment by YunYouJun** [comment]
这是 [star-markdown-css](https://github.com/YunYouJun/star-markdown-css) 的一个 BUG，`v0.4.1` 已修复。
感谢反馈，将发布于 `v1.10.7`。
